### PR TITLE
1.0.0 to Main

### PR DESCRIPTION
**Pull request by EverettRichards**
Pulling all our immense progress from _development_ onto _main_. We did it, boys! 1.0.0 in the bucket.

### CODE REVIEW

File: stocks.h2.mv.db
Comment (mingzezh) [inline stocks.h2.mv.db:1]: this file should be in gitignore

File: Server/src/main/resources/SP500in2018.csv
Comment (mingzezh) [inline Server/src/main/resources/SP500in2018.csv:1]: do we  still need this?

File: Common/src/main/mock_data/stock_symbol.json
Comment (mingzezh) [inline Common/src/main/mock_data/stock_symbol.json:1]: probably we dont need the mock data anymore

**Review by mingzezh — APPROVED** [review]
LGTM

File: Server/src/main/resources/SP500in2018.csv
Comment (EverettRichards) [reply Server/src/main/resources/SP500in2018.csv:1]: no

File: stocks.h2.mv.db
Comment (EverettRichards) [reply stocks.h2.mv.db:1]: how to do that?

File: Common/src/main/mock_data/stock_symbol.json
Comment (EverettRichards) [reply Common/src/main/mock_data/stock_symbol.json:1]: correct

**Pull request merged**